tasks.py

```diff
@@ -264,6 +264,10 @@ def install_auth_test_deps_only(c, backend):
     for b in backend:
         extra.extend(auth_backend_test_deps[b])
     c.sudo('DEBIAN_FRONTEND=noninteractive apt-get -y install ' + ' '.join(extra+auth_test_deps))
+    c.run('pwd')
+    c.run('ls -l')
+    c.run('ls -l ..')
```

Comment:
I bet this can go :)

Reply:
ah, I am too early, this debugging is still needed until the PR works. Please re-request a review when you're there.